go.mod

```diff
@@ -1,15 +1,78 @@
 module github.com/konveyor/forklift-must-gather-api
 
-go 1.15
+go 1.20
```

Comment:
so here we say that the minimum is 1.20 and then compile it with 1.19? :)

Reply:
well.. it worked locally creating the image... pretty strange indeed. i will switch to 1.19